composer.json

```diff
@@ -38,7 +38,7 @@
     ],
     "local:phpunit": [
       "wp @local db query --dbuser=root --dbpass=password \"CREATE DATABASE IF NOT EXISTS wordpress_test DEFAULT CHARSET utf8\"",
-      "vagrant ssh -c \"cd /vagrant/content/plugins/core-sitemaps && WP_TESTS_DB_PASS=password composer run test:phpunit\""
```

Comment:
Not sure we need to change this. Also doesn't this break something?

Reply:
will revert.